Description: No more use SSLv3_client_method
 It has been removed from Debian's OpenSSL package.
 .
 For forwarding it to upstream a dynamic test like for SSLv2 is
 necessary. This is just a debian-specific hard-coded decision.
Author: Axel Beckert <abe@debian.org>
Forwarded: no

Index: xymon/xymonnet/contest.c
===================================================================
--- xymon.orig/xymonnet/contest.c	2016-02-05 17:38:59.000000000 +0100
+++ xymon/xymonnet/contest.c	2016-02-05 17:38:59.000000000 +0100
@@ -482,8 +482,6 @@
 		  case SSLVERSION_V2:
 			item->sslctx = SSL_CTX_new(SSLv2_client_method()); break;
 #endif
-		  case SSLVERSION_V3:
-			item->sslctx = SSL_CTX_new(SSLv3_client_method()); break;
 		  case SSLVERSION_TLS1:
 			item->sslctx = SSL_CTX_new(TLSv1_client_method()); break;
 		  default:
